### PR TITLE
Some LoAF explainer edits for TAG review

### PR DESCRIPTION
**Pull request by noamr**

### CODE REVIEW

File: loaf-explainer.md [hunk 1]
```diff
@@ -6,21 +6,26 @@ This is work in progress. Feedback welcome, lots of things might change etc.
...
 
 Since having busy ("LoAF-heavy") sequences can potentially cause delayed response to interactions, and the LoAF entries themselves contain information about
-what was blocking, e.g. long scripts or layout, LoAF can potentially become a powerful tool, enabling the use of real-user monitoring (RUM) to diagnose
-for this type of performance issue.
+what was blocking, e.g. long scripts or layout, LoAF can potentially become a powerful tool, enabling the use of real-user monitoring (RUM) to diagnose for this type of performance issue.
+
+## Value to End Users & Publishers
+
+Several case studies like [The Economic times](https://web.dev/economic-times-inp/#how-has-improving-inp-helped-the-economic-times) and [RedBus](https://web.dev/redbus-inp/#business-impact), that improving page responsiveness via the INP metric directly helps with user delight and has a business impact.
```
Comment (clelland) [inline loaf-explainer.md:19]: I think this sentence is missing a verb -- "have shown that improving page responsiveness", or "suggest that improving page responsiveness", maybe?

File: loaf-explainer.md [hunk 2]
```diff
@@ -6,21 +6,26 @@ This is work in progress. Feedback welcome, lots of things might change etc.
...
-point in time, the browser was busy, in such a way that it took a long time from the beginning of a task until updating the rendering
-(or until the point where it was clear that no render was necessary).
+Long animation frames (LoAF), as a revamp of `longtasks`, aim to assist with that: a LoAF is an indication that in a particular point in time, the browser was congested, in such a way that it took a long time from the beginning of a task until updating the rendering (or until the point where it was clear that no render was necessary).
 
 Since having busy ("LoAF-heavy") sequences can potentially cause delayed response to interactions, and the LoAF entries themselves contain information about
-what was blocking, e.g. long scripts or layout, LoAF can potentially become a powerful tool, enabling the use of real-user monitoring (RUM) to diagnose
-for this type of performance issue.
+what was blocking, e.g. long scripts or layout, LoAF can potentially become a powerful tool, enabling the use of real-user monitoring (RUM) to diagnose for this type of performance issue.
+
+## Value to End Users & Publishers
```
Comment (clelland) [inline loaf-explainer.md:17]: I like this section, but I would switch the order of the two paragraphs in it -- start with the motivation (why INP is insufficient) and then follow up with the evidence (case studies)

File: loaf-explainer.md [hunk 3]
```diff
@@ -6,21 +6,26 @@ This is work in progress. Feedback welcome, lots of things might change etc.
...
-how their pages are doing in that regard. But INP shows you the effect, but not the cause of the jankiness.
+"Sluggishness" - the feeling that "this page is not responsive to interactions", is a common problem for users on the web today.
+By introducing [INP](https://web.dev/inp/) into Core Web Vitals, we hope that authors can have a better indication as to how their pages are doing in that regard. But INP shows you the effect, but not the cause of the sluggishness.
 
-Long animation frames (LoAF), as a revamp of `longtasks`, aim to assist with that: a LoAF is an indication that in a particular
-point in time, the browser was busy, in such a way that it took a long time from the beginning of a task until updating the rendering
-(or until the point where it was clear that no render was necessary).
+Long animation frames (LoAF), as a revamp of `longtasks`, aim to assist with that: a LoAF is an indication that in a particular point in time, the browser was congested, in such a way that it took a long time from the beginning of a task until updating the rendering (or until the point where it was clear that no render was necessary).
```
Comment (clelland) [inline loaf-explainer.md:12]: nit: *at* a particular point in time

**Review by clelland — COMMENTED** [review]
Drive-by review :)

File: loaf-explainer.md (hunk 2 above)
Comment (noamr) [reply loaf-explainer.md:17]: How is it now?

File: loaf-explainer.md (hunk 1 above)
Comment (noamr) [reply loaf-explainer.md]: Fixed

File: loaf-explainer.md [hunk 4]
```diff
@@ -6,21 +6,33 @@ This is work in progress. Feedback welcome, lots of things might change etc.
...
-for this type of performance issue.
+what was blocking, e.g. long scripts or layout, LoAF can potentially become a powerful tool, enabling the use of real-user monitoring (RUM) to diagnose for this type of performance issue.
+
+## Value to End Users & Publishers
+
+The value to end users here is enabling web pages to measure and fix sluggishness - thus making the web more responsive for everyone.
+
+See [this success story](https://www.linkedin.com/posts/erwinhofman_performance-javascript-corewebvital-activity-7122519650461958144-FFK5/) from
+a participant of the origin trial - they ran LoAF (using [RUMVision](https://www.rumvision.com/)) in user sessions, and identified that using a particular 3P JavaScript was causing sluggishness (bad INP). They refactored to not use that 3rd party, and their responsiveness
+changed dramatically.
```
Comment (yoavweiss) [inline loaf-explainer.md:23]: s/changed/improved/?

File: loaf-explainer.md [hunk 5]
```diff
@@ -6,21 +6,33 @@ This is work in progress. Feedback welcome, lots of things might change etc.
...
+
+### Business Value
+
+Several case studies like [The Economic times](https://web.dev/economic-times-inp/#how-has-improving-inp-helped-the-economic-times) and [RedBus](https://web.dev/redbus-inp/#business-impact), demonstrate that improving page responsiveness via the INP metric directly helps with user delight and has a business impact.
```
Comment (yoavweiss) [inline loaf-explainer.md:27]: s/business impact/positive business impact/?

**Review by yoavweiss — APPROVED** [review]
LGTM

**Pull request merged**